lms/static/js/student_account/utils.js

```diff
@@ -16,8 +16,8 @@
                 edxUserCookie = prefix + '-edx-user-info';
             }
 
-            var cookie = document.cookie.match('(^|;)\\s*' + edxUserCookie + '\\s*=\\s*([^;]+)');
-            var userCookie =  cookie ? cookie.pop() : $.cookie(edxUserCookie);
+            cookie = document.cookie.match('(^|;)\\s*' + edxUserCookie + '\\s*=\\s*([^;]+)');
+            userCookie = cookie ? cookie.pop() : $.cookie(edxUserCookie);
```

Comment:
1. I don't do much frontend development, so I think we should ask #fedx about this.
2. It would be great to have some sort of test coverage if possible.
3. I wonder if it makes sense to use the universal-cookie library?: https://github.com/edx/edx-platform/blob/master/package.json#L67

Reply:
I'd agree that using a library like universal-cookie is going to be preferable to doing a regex match to pull a value out of document.cookie - assuming we can use it in this file relatively easily. I expect you could add it on line 3 in this file and then use it.

Reply:
I tried to use universal-cookie in a `define` function in my own devstack and couldn't get it to work.  There may be some way to get an installed npm package to work in RequireJS in our webpack config?  I'm not quite familiar with how this works, unfortunately.

Reply:
I appreciate you tagging me. To make sure it is clear, we were giving ideas, but please don’t feel blocked by this. When you are confident in your fix, please proceed with reviews within your own team. And of course, feel free to ask questions if you need to. I just don’t want you to feel blocked. Thanks.